README.md

```diff
@@ -1,24 +1,249 @@
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Payments Creation
+> Categories Creation
+> User Authentication And Authorization
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Project is in progress and not deployed yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To start and run this project
+
+- bundle install
+- rails s
+
+To get a local copy up and running, follow these steps.
+
+- git clone https://github.com/NatanimA/PaymentsIQ.git
+
+### Prerequisites
+
+In order to run this project you don't need any prerequisites:
+```
+ruby
+```
+
+```
+postgres
+```
+```
+Nodejs
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/NatanimA/PaymentsIQ.git
+```
+
+
+### Install
+
+Install this project with:
+
+- No need of installation
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+git clone  https://github.com/NatanimA/PaymentsIQ.git
+```
+```
+bundle install
+```
+```
+rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rspec
+```
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Natanim Abesha**
+
+- GitHub: [@githubhandle](https://github.com/NatanimA)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/natanim-abesha)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Login Confirmations**
+- [ ] **Interaction of Transfers To Users**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please don't hesitate to give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank Microverse for providing all the resources so that I can build this project.
```

Comment:
- [ ] it would be better to give credit to the owner of the design as suggested in order to complete the requirements